roles/config-chrony/tasks/chrony.yml

```diff
@@ -1,10 +1,10 @@
 ---
-
-- name: "Ensure chrony service is enabled and started" 
-  service:
-    name: chronyd
-    enabled: yes
-    state: started
+# Moved this to the prereqs as it uses a loop to enable services
```

Comment:
Please just remove rather than leaving commented out

Reply:
Fixed and pushed.